.bowerrc

```diff
@@ -0,0 +1,6 @@
+{
+  "ignoredDependencies": [
+    "bootstrap",
+    "bootstrap-sass-official"
```

Comment:
I mean we can just ignore `bootstrap` in `.bowerrc`, but not with bootstrap-sass-official since it's in [their devDep](https://github.com/thomaspark/bootswatch/blob/gh-pages/bower.json#L37)